backend/kernelCI_app/views/treeDetailsView.py

```diff
@@ -89,8 +90,13 @@ def get(self, request, commit_hash):
                 table = 'builds'
             elif field in checkout_fields:
                 table = 'checkouts'
+            elif field == 'buildStatus':
+                table = 'builds'
+                field = 'valid'
+                f['value'] = f['value'] == 'Success'
```

Comment:
?

Reply:
This maps the value that we have on the front-end to the one that we use on the database